packages/auth/src/signature/middleware.ts

```diff
@@ -110,7 +111,11 @@ function validateSigInputComponents(
     !sigInputComponents.includes('@target-uri') ||
```

Comment:
I'm finally having `open-payments` http signatures verified by `auth`.
`@target-uri` needs to use `ctx.request.href` instead of
https://github.com/interledger/rafiki/blob/632bdfb15a920062637dc8a273034e58b382205b/packages/auth/src/signature/middleware.ts#L137
https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-message-signatures-09#section-2.2.3